jcl/src/java.base/share/classes/java/lang/invoke/MethodHandle.java

```diff
@@ -1284,6 +1284,13 @@ private static final MethodHandle resolveInvokeDynamic(long j9class, String name
 				throw new Error(nsme);
 			}
 		}
+
+/*[IF Java11]*/
+		/* The result of the resolution of a dynamically-computed call site must not be null. */
+		if (null == result) {
+			throw new BootstrapMethodError();
```

Comment:
Please add a message to this Error to make it clear to users why the BME is being thrown.  Something like "Bootstrap method returned null" would be sufficient.